librz/include/rz_heap_glibc.h

```diff
@@ -282,6 +281,46 @@ typedef struct rz_heap_chunk_list_item {
 	char *status; /* Status of the chunk, allocated/free/corrupted */
 } RzHeapChunkListItem;
 
+typedef struct rz_arena_list_item {
+	ut64 addr; /* Base addr of the arena */
+	char *type; /* Type of arena, main/thread */
+	MallocState *arena; /* The MallocState for the arena */
+} RzArenaListItem;
+
+typedef struct rz_heap_chunk_simple {
+	ut64 addr; /* Base addr of the chunk*/
+	ut64 prev_size; /* size of prev_chunk*/
+	ut64 size; /* size of chunk */
+	bool NMA; /* flag for NON_MAIN_ARENA */
+	bool PI; /* flag for PREV_INUSE*/
+	bool IM; /* flag for IS_MMAPPED*/
```

Comment:
Just use the full names here without caps lock, no reason to abbreviate and it's not as readable when you're accessing these fields from Cutter/other files.